youtube_dl/extractor/spankwire.py

```diff
@@ -54,62 +55,35 @@ def _real_extract(self, url):
         req = sanitized_Request('http://www.' + mobj.group('url'))
         req.add_header('Cookie', 'age_verified=1')
         webpage = self._download_webpage(req, video_id)
+        video_data = self._download_json(sanitized_Request('https://www.spankwire.com/api/video/' + video_id + '.json'), video_id)
 
-        title = self._html_search_regex(
-            r'<h1>([^<]+)', webpage, 'title')
-        description = self._html_search_regex(
-            r'(?s)<div\s+id="descriptionContent">(.+?)</div>',
-            webpage, 'description', fatal=False)
-        thumbnail = self._html_search_regex(
-            r'playerData\.screenShot\s*=\s*["\']([^"\']+)["\']',
-            webpage, 'thumbnail', fatal=False)
+        title = video_data.get('title')
+        description = video_data.get('description')
+        thumbnail = video_data.get('poster')
 
-        uploader = self._html_search_regex(
-            r'by:\s*<a [^>]*>(.+?)</a>',
-            webpage, 'uploader', fatal=False)
+        uploader = self._search_regex(
+            r'<a[^>]+class="uploaded__by"[^>]*>(.+?)</a>',
+            webpage, 'uploader', flags=re.DOTALL, fatal=False)
```

Comment:
Move flags into regex.

Reply:
@dstftw  please provide any example i will follow it. Since i am new to python i am unable to understand how to correct this.